### PR TITLE
Updated for React 0.14.*

### PR DESCRIPTION
**Pull request by olivierguerriat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-pikaday-component",
-  "version": "0.1.4",
+  "version": "0.1.5",
```
Comment (voronianski) [inline package.json]: this should be `0.2.0` as it might break someone

Comment (olivierguerriat) [inline package.json]: Right, I changed it.

**Comment by voronianski** [comment]
@olivierguerriat thanks

**Comment by voronianski** [comment]
published 0.2.0 on npm

**Comment by olivierguerriat** [comment]
Great, thank *you* for this nice wrapper!

**Pull request merged**